test/ddt_suite/validate.lua

```diff
@@ -448,6 +448,27 @@ for schema_no = 1,#records do
 
 end
 
+_G['schema_no'] = nil
+
+-- record union
+
+local foo = [[{
```

Comment:
Please, inline the schema or give it more descriptive name

Reply:
`record_union` (to match other names within the file) is okay?

Reply:
I prefer something like 
`schema_blank_nullable_field`

Reply:
'blank_nullable_field' is not a property of the schema. Bored with this, will inline.